include/odp/api/spec/packet_io_stats.h

```diff
@@ -111,6 +126,69 @@ int odp_pktio_stats(odp_pktio_t pktio, odp_pktio_stats_t *stats);
  */
 int odp_pktio_stats_reset(odp_pktio_t pktio);
 
+/**
+ * Get extra statistics counter information for pktio handle
+ *
+ * Returns the number of implementation specific packet IO extra statistics
+ * counters supported by the interface. Outputs up to 'num' extra statistics
+ * counter info structures when the 'info' array pointer is not NULL. If the
+ * return value is larger than 'num', there are more extra counters than the
+ * function was allowed to output. If the return value (N) is less than 'num',
+ * only info[0 ... N-1] have been written.
+ *
+ * The index of a counter in the 'info' array can be used to read the value of
+ * the individual counter with odp_pktio_extra_stat_counter(). The order of
+ * counters in the output array matches with odp_pktio_extra_stats().
+ *
+ * @param       pktio    Packet IO handle
+ * @param[out]  info     Array of extra statistics info structs for output
+ * @param       num      Maximum number of info structs to output
+ *
+ * @return Number of extra statistics
+ * @retval <0 on failure
+ */
+int odp_pktio_extra_stat_info(odp_pktio_t pktio,
+			      odp_pktio_extra_stat_info_t info[], uint32_t num);
```

Comment:
Application needs to compare return value and num. Compiler complains about comparison between signed and unsigned integers. Should we change type of num to int.

E.g. queue enq multi has "int num"
int odp_queue_enq_multi(odp_queue_t queue, const odp_event_t events[], int num)